doc/api/esm.md

```diff
@@ -1,6 +1,6 @@
 # ECMAScript Modules
 
-<!--introduced_in=v9.x.x-->
+<!--introduced_in=v8.5.0-->
```

Comment:
Non-blocking nit/question: Is the helpful/interesting information to include when the feature was introduced as Experimental or is it when it is introduced as Stable? I would argue Stable is what I'd expect to see and what would be useful when looking in docs. In which case, this probably should stay as is?

Reply:
I think this should use the first version in which the feature was documented regardless of the stability. Aren't docs for an experimental feature useful? If so why adding them in the first place?